modules/monitoring-services.nix

```diff
@@ -25,6 +25,11 @@ in {
         default = true;
       };
 
+      enableWireguard = mkOption {
+        type = types.bool;
+        default = false;
```

Comment:
True?

Reply:
this is enabled in the env files, when wireguard is needed